examples/fuse_examples/imaging/classification/knight/baseline/fuse_baseline.py

```diff
@@ -25,6 +25,11 @@
 from fuse.utils.rand.seed import Seed
 import logging
```

Comment:
You can remove the import to fuse.dl.managers and the remove the entire managers folder.